src/View/SSViewer_DataPresenter.php

```diff
@@ -316,4 +276,110 @@ public function __call($name, $arguments)
             return parent::__call($name, $arguments);
         }
     }
+
+    /**
+     * Evaluate a template override
+     *
+     * @param string $property Name of override requested
+     * @param array $overrides List of overrides available
+     * @return null|array Null if not provided, or array with 'value' or 'callable' key
+     */
+    protected function processTemplateOverride($property, $overrides)
+    {
+        if (!isset($overrides[$property])) {
+            return null;
+        }
+
+        // Detect override type
+        $override = $overrides[$property];
+
+        // Late-evaluate this value
+        if (is_callable($override)) {
+            $override = $override();
+
+            // Late override may yet return null
+            if (!isset($override)) {
+                return null;
+            }
+        }
+
+        return [ 'value' => $override ];
+    }
+
+    /**
+     * Determine source to use for getInjectedValue
+     *
+     * @param string $property
+     * @return array|null
+     */
+    protected function getValueSource($property)
+    {
+        // Check for a presenter-specific override
+        $overlay = $this->processTemplateOverride($property, $this->overlay);
+        if (isset($overlay)) {
```

Comment:
Can’t these just be `if ($overlay)`?

Reply:
The template could be a valid empty string, so I use nulls as explicit non-value.